readme.md

```diff
@@ -81,27 +81,54 @@ to [run the E2E test](./LOCAL_TESTING.md).
 [Discord](https://discord.com/invite/zetachain) |
 [Telegram](https://t.me/zetachainofficial) | [Website](https://zetachain.com)
 
+## Releases
 
-## Creating a Release Candidate
-Creating a release candidate for testing is a straightforward process. Here are the steps to follow:
+To start a new major release, begin by creating and pushing a `release/` branch based on `develop`.
```

Comment:
I think can also delete current `main` branch and rename `develop` into `main` since in our current flow we don't use main